todo-frontend-angular/src/app/pages/welcome/welcome.component.ts

```diff
@@ -0,0 +1,15 @@
+import { Component, OnInit } from '@angular/core';
```

Comment:
If these are not going to be used, remove them

Reply:
Welcome page and sidemenu removed